packages/frontend/pages/oauth2/authorize.vue

```diff
@@ -92,21 +92,24 @@ watchEffect(() => {
 
 <template>
   <div>
-    <AuthorizeError v-if="!validResponseType" error="Invalid response type" />
+    <!-- <div v-if="pending">
+      Loading...
+    </div>
+    <LoginView v-else-if="error" @login="refresh" />
+    <AuthorizeError v-else-if="!validResponseType" error="Invalid response type" />
     <AuthorizeError v-else-if="!validScope" error="Invalid scope" />
     <AuthorizeError v-else-if="!validClient" />
     <AuthorizeError v-else-if="!validRedirect" error="Invalid redirect" />
-    <div v-else-if="pending">
-      Loading...
-    </div>
-    <form v-else-if="!error" @submit.prevent="authorize">
-      <p>An external application wants to access your account</p>
+  -->
+    <form>
+      <!-- <p>An external application wants to access your account</p>
       <button type="button" @click="cancel">
         Cancel
       </button>
       <button class="bg-green" type="submit">
         Authorize
-      </button>
+      </button> -->
+      <AuthorizeView username="User123" applicationName="SketchyApp" :claims="['identity', 'phone', 'address']"/>
```

Comment:
You also removed any authorize errors, which should still be present, even though their design is not implemented. I'll also need you to either move stuff from AuthorizeView here, or add events to be passed from AuthorizeView on click.